src/bitcoin-cli.cpp

```diff
@@ -122,11 +107,7 @@ static int AppInitRPC(int argc, char* argv[])
         fprintf(stderr, "Error: %s\n", e.what());
         return EXIT_FAILURE;
     }
-    if (GetBoolArg("-rpcssl", false))
-    {
-        fprintf(stderr, "Error: SSL mode for RPC (-rpcssl) is no longer supported.\n");
```

Comment:
This has been turned into an `Error parsing program options: unrecognized option 'rpcssl'` .

It might be helpful to record the unsupported options that have been turned into different errors by this patch and record them in the PR text. I noticed 'rpcssl', 'socks', 'debugnet', 'tor', 'benchmark', 'whitelistalwaysrelay' so far.